lib/locales/fr.ts

```diff
@@ -272,6 +272,8 @@ export const locale: Locale = {
     damasus_i_pope: 'Saint Damase Ier, pape († 384)',
     damien_de_veuster_priest: 'Saint Père Damien, prêtre et Missionnaire Picpus († 1889)',
     daniel_brottier_priest: 'Binheureux Daniel Brottier, prêtre, apôtre des Orphelins d’Auteuil († 1936 à Paris)',
+    dedication_of_consecrated_churches:
+      'Dédicace des églises consacrées dont on ne connaît pas la date de consécration', // src: missal_fr_2021_ed3
```

Comment:
Just a nitpick: we might change this, however, everywhere I use `mr_${lang}_${year}_${edition}` format.

```suggestion
      'Dédicace des églises consacrées dont on ne connaît pas la date de consécration', // src: mr_fr_2021_ed3
```

Reply:
I guess "mr" stands for "Missale Romanum" ? I wasn't sure, but ok to use this abbreviation 👍👍

Reply:
Yeah, `mr` stands for _Missale Romanum_.

However, it could be ambiguous with `Martyrologium Romanum`, which I would shorten to `mar` (as in _MArtyrologium Romanum_); then we could have `mir` for _MIssale Romanum_, however, I still like `mr` better. YMMV.